muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/announcer/AnnouncerJsPluginMessages_fi.properties

```diff
@@ -4,15 +4,17 @@ plugin.announcements.pageTitle = Tiedotteet
 plugin.announcer.pageTitle = Tiedotin
 plugin.announcer.link.edit = Muokkaa
 plugin.announcer.link.delete = Poista
-plugin.announcer.errormessage.loadAnnouncements = Tiedotteiden lataus ep�onnistui. Yrit� ladata sivu uudelleen.
-plugin.announcer.errormessage.loadAnnouncement = Tiedotteen lataus ep�onnistui. Yrit� ladata sivu uudelleen.
-plugin.announcer.errormessage.updateAnnouncement = Tiedotteen muokkaus ep�onnistui. Yrit� uudelleen.
-plugin.announcer.errormessage.deleteAnnouncement = Tiedotteen poisto ep�onnistui. Yrit� uudelleen.
-plugin.announcer.errormessage.createAnnouncement = Tiedotteen luominen ep�onnistui. Yrit� uudelleen.
+plugin.announcer.announcement.empty = Ei valittua tiedotusta tai tiedotusta ei l�ytynyt.
+plugin.announcer.errormessage.announcementNotAvailable = Kyseist� tiedotusta ei ole saatavilla tai sit� ei ole olemassa.
+plugin.announcer.errormessage.loadAnnouncements = Tiedotteiden lataus ep�onnistui. Yrit� ladata sivu uudelleen.
+plugin.announcer.errormessage.loadAnnouncement = Tiedotteen lataus ep�onnistui. Yrit� ladata sivu uudelleen.
+plugin.announcer.errormessage.updateAnnouncement = Tiedotteen muokkaus ep�onnistui. Yrit� uudelleen.
+plugin.announcer.errormessage.deleteAnnouncement = Tiedotteen poisto ep�onnistui. Yrit� uudelleen.
+plugin.announcer.errormessage.createAnnouncement = Tiedotteen luominen ep�onnistui. Yrit� uudelleen.
 plugin.announcer.errormessage.createAnnouncement.missing.caption = Tiedotteen otsikko on pakollinen tieto
-plugin.announcer.errormessage.createAnnouncement.missing.content = Tiedotteen sis�lt� on pakollinen tieto
-plugin.announcer.errormessage.createAnnouncement.missing.startDate = Tiedotteen alkamisp�iv�m��r� on pakollinen tieto
-plugin.announcer.errormessage.createAnnouncement.missing.endDate = Tiedotteen loppumisp�iv�m��r� on pakollinen tieto
+plugin.announcer.errormessage.createAnnouncement.missing.content = Tiedotteen sis�lt� on pakollinen tieto
```

Comment:
Encoding problems. Might be because editing those files in vscode and vscode tries to handle them in utf8 encoding or something similar which is wrong.

Locale files should be using ISO-8859-1 encoding.